docs/course_team/concepts/individual_learner_dashboard.rst

```diff
@@ -0,0 +1,16 @@
+Individual Learner Dashboard Reports
```

Comment:
This one should be moved too, it's linked to: https://docs.openedx.org/projects/openedx-aspects/page/reference/individual_learner_dashboard.html